core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.html

```diff
@@ -0,0 +1,89 @@
+<p-overlayPanel #op styleClass="dotDeviceSelector__overlayPanel">
+    <div class="dotDeviceSelector__column">
+        <div class="dotDeviceSelector__panel">
+            <div class="dotDeviceSelector__header">
+                <h3 class="dotDeviceSelector__header-title">
+                    {{ 'editpage.device.selector.title' | dm }}
+                </h3>
+                <p-button icon="pi pi-plus" styleClass="p-button-rounded p-button-text"></p-button>
+            </div>
+            <div class="dotDeviceSelector__content">
+                <ul class="device-list">
+                    <li
+                        class="device-list__item"
+                        *ngFor="let option of defaultOptions"
+                        (click)="change(option)"
```

Comment:
`<li>` are not clickables elements this have a11y issues. More info: https://chat.openai.com/share/70bfe9b5-7897-4210-ad20-9a2c10e5f0d5